#input
float64[] x_coords
float64[] y_coords
float64[] phi_vals
---
#result
bool completed
---
#feedback
int32 steps_completed
